bridges/snowbridge/primitives/inbound-queue/src/v2/converter.rs

```diff
@@ -310,15 +331,9 @@ where
 
 		let bridge_owner = Self::get_bridge_owner()?;
```

Comment:
In rust, the idiom is that getters don't start with `get_`.

```suggestion
		let bridge_owner = Self::bridge_owner()?;
```

Reply:
[342dd0dc6f8480c63fa4b818dca0d6dba97be1bf](https://github.com/vgeddes/polkadot-sdk/pull/32/commits/342dd0dc6f8480c63fa4b818dca0d6dba97be1bf)